index.html

```diff
@@ -39,7 +39,8 @@
 		  
           };
     </script>
-<link rel="stylesheet" data-import href="https://w3c.github.io/i18n-drafts/style/respec_2022.css">
+ 
+<!--link rel="stylesheet" data-import href="https://w3c.github.io/i18n-drafts/style/respec_2022.css"-->
```

Comment:
is this supposed to be:

```
<style data-include="https://w3c.github.io/i18n-drafts/style/respec_2022.css"
```

Reply:
@xfq @r12a can I get a look at this one? It's the only thing preventing me from pushing to TR